libs/causality/src/lib/CausalAnalysisDashboard/Controls/CausalAnalysisView/CausalIndividualView/CausalIndividualChart.tsx

```diff
@@ -141,6 +146,7 @@ export class CausalIndividualChart extends React.PureComponent<
               canDither={this.state.chartProps.chartType === ChartTypes.Scatter}
               onAccept={this.onXSet}
               onCancel={this.setXClose}
+              telemetryHook={this.props.telemetryHook}
```

Comment:
I see telemetryHook is passed in many files. Curious if it is possible to have context for this?

Reply:
Good point!! Created a task for tracking: [Task 1893214](https://msdata.visualstudio.com/Vienna/_workitems/edit/1893214): Create a context for telemetryHook